test/cpp/test_aten_xla_tensor.cpp

```diff
@@ -4244,7 +4244,6 @@ TEST_F(AtenXlaTensorTest, TestUpsampleNearest2DWithScale) {
       AllClose(result, xla_result);
     });
   }
-  ExpectCounterNotChanged("aten::.*", cpp_test::GetIgnoredCounters());
```

Comment:
why do we remove this line? This seems like a regression.

Reply:
Thanks for the comment. For some of these tests, I noticed that they fallback to CPU through this if statement here: https://github.com/pytorch/xla/blob/3bd2d21ee6874010c95de2f2d2e6660afc47f369/torch_xla/csrc/aten_xla_type.cpp#L2921

In the removed `upsample_nearest2d.vec` XLANativeFunctions, they did not have such fall back so `aten::` was not invoked. 

Do you know why we have this fallback in the current `umsample_nearest` function?

Reply:
hmm, check `GetOutputSizeWithScale`.

For `upsample_nearest2d.vec`, both `scale` and `output_size` are optional and user should only specify one. If output is specified, we check scale is not set. If `output` is not set, we use `scale` and input to compute the `output_size`.

For `upsample_nearest2d`, `output_size` is not optional, hence we can't support scale at the same time. 

For this test, it seems like we provide the scale, but we don't provide the `output_size` (we passed `c10::nullopt`), I guess we need to look into `upsample_nearest2d`'s `output_size`, upstream must pass us something.

Reply:
Thanks for the explanation. Looking back into the upstream PR, the op that we need to remove is only `upsample_nearest2d_backward.vec` op. We should keep the `upsample_nearest2d.vec` op, as upstream also keeps it. Updated the PR to reflect this.